mscore/scoreview.h

```diff
@@ -153,7 +153,8 @@ class ScoreView : public QWidget, public MuseScoreView {
       void paint(const QRect&, QPainter&);
 
       void objectPopup(const QPoint&, Element*);
-      void measurePopup(const QPoint&, Measure*);
+//      void measurePopup(const QPoint&, Measure*);
```

Comment:
better delete, git history is enough to show it was once there

Reply:
This isn't actually my change.  Not sure how it got here.